x-pack/plugins/apm/public/components/app/ErrorGroupOverview/view.js

```diff
@@ -8,12 +8,13 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { get } from 'lodash';
 import { HeaderContainer } from '../../shared/UIComponents';
-import TabNavigation from '../../shared/TabNavigation';
+import { TransactionErrorTabs } from '../../shared/TransactionErrorTabs';
```

Comment:
This file will go away once the stuff in beta.tsx is finalized. That TransactionErrorTabs file is gone as I opted to render the tabs in the service details page instead of sharing them across pages.